tests/integration/orcid/cassettes/test_push_record_with_orcid_new.yaml

```diff
@@ -1,28 +1,4 @@
 interactions:
-- request:
```

Comment:
I was a bit confused by this at first, but it’s OK: this is the part that does the request against inspire to get the bibtex. The fact the bibtex still gets sent is further down on L21-L32.